### PR TITLE
Alphabetical check in Tests

### LINKED ISSUE #334: Add alphabetical check to the CI tests
_Originally posted by @swapagarwal in https://github.com/swapagarwal/swag-for-dev/pull/331#discussion_r329973678_

### PR DESCRIPTION
**Pull request by jnnck**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #334 

New to this pull request thing, so open to all feedback!

### CODE REVIEW

**Comment by aslafy-z** [comment]
I edited your initial comment to include "Fixes #ISSUE_NB" in order to ask github to automatically close the issue when this PR is merged.

LGTM, but you still have to update the package-lock.json (it should be done by `npm install`).

**Comment by vikaspotluri123** [comment]
Please wait for my review before merging later 😄

**Comment by jnnck** [comment]
Included lock file! Thanks for your help en comments!

File: data.json
Comment (aslafy-z) [inline data.json]: This change is out of the scope of this PR. Can you revert it? Thank you!

**Comment by jnnck** [comment]
No idea how that got in here, seems to have been added in #327 and removed again in #323

File: test/functional/data.spec.js
Comment (vikaspotluri123) [inline test/functional/data.spec.js]:
```suggestion
		const left = data.map(({name}) => name.toUpperCase());
		const right = [...left].sort();

		expect(left).to.deep.equal(right);
```

I prefer not to add dependencies for trivial things. Since this dep is only used once, adding a couple of extra lines of logic is preferred over the dep.

Comment (swapagarwal) [reply test/functional/data.spec.js]: Ping @jnnck

**Review by aslafy-z — APPROVED** [review]
LGTM

**Pull request merged**
